analysis_module/mockserver.py

```diff
@@ -0,0 +1,175 @@
+import os
+import json
+import requests
+import logging
+from random import shuffle
+from math import ceil
+from sklearn.feature_extraction.text import CountVectorizer
+
+logging.getLogger().setLevel(logging.INFO)
+
+
+def get_entries_data(url):
+    try:
+        response = requests.get(url)
+        entries_data = json.loads(response.text)
+        return entries_data
+    except Exception as e:
+        raise e
+
+
+def save_data_local(dir_name, client_id, data):
+    parent_dirpath = f"/tmp/{dir_name}"
+    if not os.path.exists(parent_dirpath):
+        os.makedirs(parent_dirpath)
+    filepath = parent_dirpath + f"/{client_id}.json"
```

Comment:
It's better to use `os.path.join`, or even `Path` which allowes something like `path1 / filename`